### PR TITLE
base installation on mambaforge, skip when anaconda/miniconda is found

### LINKED ISSUE #8: update to use mambaforge as base
With 8.0.0 we also change our documentation that an installation should be based on mambaforge

### PR DESCRIPTION
**Pull request by ReimarBauer**
I have tested it for windows on a guacamole remote system.  And on linux :)

We should try this with some other systems before merging.

closes #8

### CODE REVIEW

**Review by joernu76 — APPROVED** [review]

File: .github/workflows/test_installer.yml [hunk 1]
```diff
@@ -1,75 +1,181 @@
...
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Install conda+MSS
+      - name: Install mambaforge+MSS
         run: |
-          sudo rm -rf $(which conda)
+          rm -rf $(which mamba)
```
Comment (joernu76) [inline .github/workflows/test_installer.yml:23]: I do not understand the intent of this. Simply removing the binary seems like a bad idea, in general. Can you8 add a comment about the intent of this, as it is not obvious, at least to me?
In particular as this is a fresh Ubuntu, mamba should not be present?

File: .github/workflows/test_installer.yml [hunk 2]
```diff
@@ -1,75 +1,181 @@
...
           cd $GITHUB_WORKSPACE
           chmod +x LinuxMac.sh
           ./LinuxMac.sh -a
       - name: Test MSS installed
         run: |
-          . /home/runner/miniconda3/bin/activate mssenv
-          msui --version | grep "Mission Support System (MSS)"
+          /home/runner/mambaforge/envs/mssenv/bin/msui --version | grep "Mission Support System (MSS)"
```
Comment (joernu76) [inline .github/workflows/test_installer.yml:29]: Does this activate the environment? We had some issues in th past, with, e.g. pyproj and un-activated environemnts.

File: LinuxMac.sh [hunk 3]
```diff
@@ -20,53 +21,93 @@
...
+    . "$HOME/mambaforge/etc/profile.d/conda.sh" &&
+    . "$HOME/mambaforge/etc/profile.d/mamba.sh" &&
+    mamba init &&
+    mamba update -n base mamba && # update mamba to the newest version
+    mamba activate mssenv || {
     echo "mssenv not found, creating..." &&
-    conda create -n mssenv mamba -y &&
-    conda activate mssenv || { echo "Environment not found, aborting"; exit 1; }
+    mamba create -n mssenv -y &&
+    mamba activate mssenv || { echo "Environment not found, aborting"; exit 1; }
```
Comment (joernu76) [inline LinuxMac.sh:103]: -> 'mssenv' environment could not be created, aborting

File: .github/workflows/test_installer.yml (hunk 1 above)
Comment (ReimarBauer) [reply .github/workflows/test_installer.yml:23]: lets check this, I think I kept it because it was there.

**Review by joernu76 — APPROVED** [review]

**Pull request merged**